week-1-ts-ecosystem/w01e04-light-me-up/src/powerSource.ts

```diff
@@ -0,0 +1,11 @@
+export class PowerSource {
+  private energySupply: number = 100;
```

Comment:
taki zapis `:number  = 100`

Unikaj. Ponieważ to przecież typ prosty.
Musisz go zaincjować więc jak dajesz `= 100;` to Ty sam widzisz że to `number`

A nie musisz tego mówić `TypeScript`'owi - bo on to sam wnioskuje :)

Reply:
Faktycznie, usunąłem deklarację prostego typu.